examples/high_frequency/radiofrequency_mmwave/coplanar_waveguide.py

```diff
@@ -140,9 +162,10 @@
 
 # +
 sm_obj_list = []
-ids = [1, 2, 3]
-if AEDT_VERSION >= "2023.1":
-    ids = [0, 1, 2]
+ids = [0, 1, 2]
+
+if AEDT_VERSION < "2023.1":  # Support old versions of AEDT.
+    ids = [1, 2, 0]
```

Comment:
```suggestion
    ids = [1, 2, 3]
```
Shouldn't that be `[1,2,3]` like before ?